docs/galaxy_releases.rst

```diff
@@ -0,0 +1,219 @@
+******************************
+Ansible Galaxy Release Process
+******************************
+
+.. contents:: Topics
+
+Overview
+========
+In this document we will cover common recommended ways to automate your Ansible
+Collection release process.
+
+**Warning:** Subject to frequent updates
+       This is a "living document", expect it to change as we progress with the
+       Collections work over the next few months.
+
+GitHub Action Workflow for Collection Release
+=============================================
+
+Ansible Galaxy does not allow us to use the web-hook we have grown accustom to
+for publishing or releasing an Ansible Collection. To publish your Ansible
+Collection to GitHub you have two options:
+
+#.  Uploading the :code:`tar.gz` that was created during the build command
+    :code:`ansible-galaxy collection build`
+#.  By using the :code:`ansible-galaxy collection publish` command after the
+    build command.
+
+Ansible Collection Build Process
+--------------------------------
+
+
+Create the :code:`build/` directory.
+::
+
+  build
+  ├── galaxy_deploy.yml
+  └── templates
+      └── galaxy.yml.j2
+
+Instead of having the :code:`galaxy.yml` file in our root, we will be generating
+this each time we create a release.
+
+You will need to create the :code:`galaxy_deploy.yml` Ansible playbook which
+will handle the build and publish steps for us. It's contents should look
+something like this.
+
+galaxy_deploy.yml
+^^^^^^^^^^^^^^^^^
+.. code-block:: yaml
+
+  ---
+  - hosts: localhost
+    connection: local
+    gather_facts: false
+    vars:
+      # This should be set via the command line at runtime.
+      tag: "{{ github_tag.split('/')[-1] }}"
+    pre_tasks:
+      - name: Ensure the ANSIBLE_GALAXY_TOKEN environment variable is set.
+        fail:
+          msg: ANSIBLE_GALAXY_TOKEN is not set.
+        when: "lookup('env','ANSIBLE_GALAXY_TOKEN') | length == 0"
+      - name: Ensure the ~/.ansible directory exists.
+        file:
+          path: ~/.ansible
+          state: directory
+      - name: Write the Galaxy token to ~/.ansible/galaxy_token
+        copy:
+          content: |
+            token: {{ lookup('env','ANSIBLE_GALAXY_TOKEN') }}
+          dest: ~/.ansible/galaxy_token
+          mode: '0600'
+    tasks:
+      - name: Template out the galaxy.yml file.
+        template:
+          src: templates/galaxy.yml.j2
+          dest: ../galaxy.yml
+        register: galaxy_yml
+      - name: Build the collection. # noqa 503
+        command: >
+          ansible-galaxy collection build
+          chdir=../
```

Comment:
this chdir thing looks weird. did you mean something like
```suggestion
        args:
          chdir: ../
```

Reply:
No, what I pasted actually works, @geerlingguy used it and I used it that same way

Reply:
I believe that it works. But my concern here is readability.